public/app/plugins/datasource/tempo/traceql/TempoQueryBuilderOptions.tsx

```diff
@@ -31,162 +32,174 @@ const parseIntWithFallback = (val: string, fallback: number) => {
+                  placeholder="auto"
+                  type="number"
+                  min={1}
+                  defaultValue={query.spss || DEFAULT_SPSS}
+                  onCommitChange={onSpssChange}
+                  value={query.spss}
+                />
+              </EditorField>
+              <EditorField label="Table Format" tooltip="How the query data should be displayed in the results table">
+                <RadioButtonGroup
+                  options={[
+                    { label: 'Traces', value: SearchTableType.Traces },
+                    { label: 'Spans', value: SearchTableType.Spans },
+                  ]}
+                  value={query.tableType}
+                  onChange={onTableTypeChange}
+                />
+              </EditorField>
+              <EditorField label="Streaming" tooltip={<StreamingTooltip />} tooltipInteractive>
+                <div>{searchStreaming ? 'Enabled' : 'Disabled'}</div>
+              </EditorField>
+            </QueryOptionGroup>
+          )}
+
+          <QueryOptionGroup
+            title="Metrics Options"
+            collapsedInfo={collapsedMetricsOptions}
+            isOpen={isOpen}
+            onToggle={toggleOpen}
           >
-            <AutoSizeInput
-              className="width-4"
-              placeholder="auto"
-              type="string"
-              defaultValue={query.step}
-              onCommitChange={onStepChange}
-              value={query.step}
-            />
-          </EditorField>
-          <EditorField label="Type" tooltip="Type of metrics query to run">
-            <RadioButtonGroup
-              options={[
-                { label: 'Range', value: MetricsQueryType.Range },
-                { label: 'Instant', value: MetricsQueryType.Instant },
-              ]}
-              value={query.metricsQueryType}
-              onChange={onMetricsQueryTypeChange}
-            />
-          </EditorField>
-
-          <EditorField label="Streaming" tooltip={<StreamingTooltip />} tooltipInteractive>
-            <div>{metricsStreaming ? 'Enabled' : 'Disabled'}</div>
-          </EditorField>
-          {/*<EditorField*/}
-          {/*  label="Exemplars"*/}
-          {/*  tooltip="Defines the amount of exemplars to request for metric queries. A value of 0 means no exemplars."*/}
-          {/*>*/}
-          {/*  <AutoSizeInput*/}
-          {/*    className="width-4"*/}
-          {/*    placeholder="auto"*/}
-          {/*    type="string"*/}
-          {/*    defaultValue={query.exemplars}*/}
-          {/*    onCommitChange={onExemplarsChange}*/}
-          {/*    value={query.exemplars}*/}
-          {/*  />*/}
-          {/*</EditorField>*/}
-        </QueryOptionGroup>
-      </div>
-    </EditorRow>
-  );
-});
+            {!isAlerting && (
+              <EditorField
+                label="Step"
+                tooltip="Defines the step for metric queries. Use duration notation, for example 30s or 1m"
```

Comment:
Out of curiosity - why we do not allow to change this value? It doesn't seem to be read-only for Prom queries 🤔 

Also when I open the editor (on the ephemeral) the value is filled with `30s`? How is it being set? Should we document it somehow?

<img width="375" alt="Screenshot 2025-05-19 at 13 13 43" src="https://github.com/user-attachments/assets/4771a451-2c71-4017-84fb-313dcc381800" />

Reply:
Step has been removed